doc/reference/cardano-node-cli-reference.md

```diff
@@ -11,7 +11,7 @@ The set of `cardano-cli shelley` commands include:
 * `transaction`: Shelley transaction commands
 * `node`: Shelley node operation commands
 * `stake-pool`: Shelley stake pool commands
-* `query`: Shelley node query commands. This queries the local node whose Unix domain socket is obtained from the CARDNAO_NODE_SOCKET-PATH environment variable. 
+* `query`: Shelley node query commands. This queries the local node whose Unix domain socket is obtained from the CARDNAO_NODE_SOCKET-PATH environment variable.
```

Comment:
`CARDANO` is misspelt.

Also, can you clarify if the environment variable is `CARDNAO_NODE_SOCKET-PATH` or `CARDNAO_NODE_SOCKET`?

If the latter, I recommend monospacing  `CARDANO_NODE_SOCKET` and adding a space to either side of the hyphen.